src/x/checked/bytes_test.go

```diff
@@ -60,6 +62,22 @@ func TestBytes(t *testing.T) {
 
 	finalizerCalls := 0
 	onFinalize = func(finalizing Bytes) {
+		// Ensure that RefCount.Finalize was called before the bytesRef finalizer.
+		stack := debug.Stack()
+		var (
+			refCountFinalizeLineNumber   int
+			bytesRefOnFinalizeLineNumber int
+		)
+		for idx, line := range strings.Split(string(stack), "\n") {
+			if strings.Contains(line, "(*bytesRef).OnFinalize") {
+				bytesRefOnFinalizeLineNumber = idx
+			}
+			if strings.Contains(line, "(*RefCount).Finalize") {
+				refCountFinalizeLineNumber = idx
+			}
+		}
+		assert.True(t, refCountFinalizeLineNumber > bytesRefOnFinalizeLineNumber)
```

Comment:
Want to also test that neither of them are default values (i.e. they both appear)?

Reply:
Maybe set them both to `-1` and verify `> -1` as well?